week-1-practise/level-1/10-function-output-3.js

```diff
@@ -1,6 +1,6 @@
 function concatenate(firstWord, secondWord, thirdWord) {
-    // Write the body of this function to concatenate three words together
+  return firstWord + secondWord + thirdWord; // Write the body of this function to concatenate three words together
```

Comment:
Nice, that works. Can you find a string method that lets you do the same? MDN documentation or just Google might help :)